osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -40,6 +41,8 @@ public class SkipOverlay : CompositeDrawable, IKeyBindingHandler<GlobalAction>
 
         private bool isClickable;
 
+        public BindableBool IsSkippable = new BindableBool()!;
```

Comment:
Should be exposed as an `IBindable<bool>` (search in solution for many examples).

Also I don't know what the `!` is doing there, but it shouldn't exist.